.eslintrc.js

```diff
@@ -1,15 +1,3 @@
-const path = require('path');
-const base = require('@backstage/cli/config/eslint');
-
 module.exports = {
-  ...base,
-  rules: {
-    ...base.rules,
-    'notice/notice': [
-      'error',
-      {
-        templateFile: path.resolve(__dirname, 'scripts/copyright.js'),
-      },
-    ],
-  },
+  extends: ['./packages/cli/config/eslint.js', '@spotify/eslint-config-oss'],
```

Comment:
Can this still be `@backstage/cli/config/eslint`? Better as a reference for standalone apps that don't have the local cli package.

Reply:
agreed - I didn't think of the best way to do that at first, but I just pushed an update I think satisfies you while also avoiding a lot of deeply-nested spreads and leans more comfortably on the `extend` feature. lmk if that works!

Reply:
tl;dr: `require.resolve('@backstage/cli/config/eslint.js')`